qiita_pet/util.py

```diff
@@ -52,7 +52,7 @@ def clean_str(item):
     str
         cleaned string
     """
-    return str(item).replace(" ", "_").replace(":", "")
+    return str(item).replace(" ", "_").replace(":", "").replace(".", "")
```

Comment:
Yup. I'm sure it doesn't change anything, as this function is for cleaning out string items that would screw up jquery use, so having a period in the text would definitely screw up a jquery check by class. I can update the docstring to reflect this.